ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.jsonschema.json

```diff
@@ -10617,6 +10617,22 @@
                 "^_.+$": {}
               },
               "title": "Multicast Replication"
+            },
+            "mdb": {
+              "type": "object",
+              "description": "MDB Configuration",
+              "properties": {
+                "profile": {
+                  "type": "string",
+                  "description": "Set MDB profile",
+                  "title": "Profile"
+                }
+              },
+              "additionalProperties": false,
+              "patternProperties": {
+                "^_.+$": {}
+              },
+              "title": "Mdb"
```

Comment:
please add MDB as an acronym in `ansible_collections/arista/avd/plugins/plugin_utils/schema/key_to_display_name.py`